src/MOF/mof.schema.json

```diff
@@ -0,0 +1,506 @@
+{
+    "$schema": "https://json-schema.org/schema#",
+    "$id": "https://github.com/odow/MathOptFormat.jl/tree/master/src/MOF/mof.schema.json",
+    "title": "The schema for MathOptFormat",
+    "type": "object",
+    "properties": {
+        "version": {
+            "const": 0
+        },
+        "author": {
+            "description": "The author of the model for citation purposes",
+            "type": "string"
+        },
+        "description": {
```

Comment:
Is there a name field on the model? I.e. corresponding to http://www.juliaopt.org/MathOptInterface.jl/dev/apireference/#MathOptInterface.Name.